Header header

string decision_maker
DecisionState state
HeuristicDecision decision